Adding a note #1596 (#1607)
MicrosoftDocs/OfficeDocs-SharePoint: OneDrive/control-access-to-mobile-app-features.md

# Control access to features in the OneDrive and SharePoint mobile apps

If your organization has **Microsoft Intune** or **Enterprise Mobility + Security**, you can use the OneDrive admin center to create a global policy that manages the OneDrive and SharePoint mobile apps for Android and iOS. This policy only applies to users in your organization who are licensed for Microsoft Intune or Enterprise Mobility + Security.
  

>  >[!NOTE]

Reviewer: ```suggestion
 > [!NOTE]
```

>  >Your admin account needs to have an Intune license assigned to it for you to change the mobile application management settings in the OneDrive admin center. 

Reviewer: ```suggestion
 >Your admin account needs to have an Intune license assigned to it for you to change the mobile application management settings in the OneDrive admin center. 
```

>  >[!NOTE]

Reviewer: ```suggestion
 > [!NOTE]
```

>  > In the case that you disable a policy, this won't remove it from Intune.

Reviewer: ```suggestion
 > Disabling a policy won't remove it from Intune.
```
